example/package.json

```diff
@@ -11,12 +11,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@babel/polyfill": "^7.4.4",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-node-resolve": "^13.1.3",
     "es6-object-assign": "^1.1.0",
-    "rollup": "^1.16.3",
-    "rollup-plugin-commonjs": "^10.0.1",
-    "rollup-plugin-node-builtins": "^2.1.2",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-node-resolve": "^5.2.0"
+    "rollup": "^2.67.0"
```

Comment:
Unfortunately yes, because https://github.com/JRJurman/rollup-plugin-polyfill/blob/e063376496e0328169001d74e70b19156d3f908c/src/rollup-plugin-polyfill.js#L40 requires https://github.com/rollup/rollup/releases/tag/v2.67.0.
Beyond that, we rely on the default export detection in https://github.com/rollup/rollup/releases/tag/v2.66.0.

Reply:
Understood. It took me by surprise to have a conflict when I was running 2.66.0 already. 👍